### ISSUE TITLE
a11y: modal dialogs cannot be escaped via the keyboard

### ISSUE THREAD

**Issue by matthew-hagemann** [canonical/desktop-security-center#129]
Once you start tabbing through the keyboard options in a modal dialog, you have no way to escape the modal dialog.

I'd expect either ESC to close it, or to be able to tab up to the X button to close it.
